ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -10,7 +10,7 @@ apply plugin: 'maven-publish'
  */
 ext {
     // String property to store version name.
-    stringVersion = "6.8.1.0"
+    stringVersion = "6.9.0.0"
```

Comment:
can the version string be "6.9.0.0-early1" and making sure it doesnt break anything, do we ever pass this version downstream to adapters ( dont think so but making sure)

Reply:
@palrahul we can change this, but its used by us to build VersionInfo for admob and we'll have to change it back when we cut another release and it won't parse correctly. I'd recommend we leave it for now, and look to fixing it in the future.